tests/test_edf.py

```diff
@@ -978,3 +949,10 @@ def test_writing_edfplus_works_with_data_record_duration_different_from_1(
         data_record_duration=data_record_duration,
     )
     edf.to_bytes()
+
+
+def test_rounding_of_physical_range_does_not_produce_clipping_or_integer_overflow():
+    data = np.array([0, 0.0000014999])
+    sig = EdfSignal(data, 1)
+    np.testing.assert_allclose(sig.data, data, atol=1e-11)
+    assert sig._digital.tolist() == [-32768, 16380]
```

Comment:
Could add a comment to explain the exected value of 16380, maybe simply as a formula:
`# round( (0.0000014999 - 0.0) / 0.000002 * 65535 + (-32768) ) = 16380`

Reply:
Great idea, added!